src/components/CreationFormCard.jsx

```diff
@@ -5,258 +5,271 @@ import CardActions from "@mui/material/CardActions";
-                                    inputProps: { min: 1, max: 10 },
-                                    style: { color: 'white' },
-                                }}
-                                variant="outlined"
-                                size="small"
-                            />
-                        </Grid>
+                <Grid container spacing={2} alignItems="center">
+                    <Grid item xs={2}>
+                        <Typography variant="body1" gutterBottom>
+                            Luck
+                        </Typography>
                     </Grid>
-                </CardContent>
-                <CardActions>
-                    <Button type="submit" variant="contained" color="orange">Generate</Button>
-                </CardActions>
-            </Card>
-        </form>
+                    <Grid item xs={8}>
+                        <Slider
+                            id="luck"
+                            name="luck"
+                            value={values.luck}
+                            onChange={handleChange}
+                            valueLabelDisplay="auto"
+                            min={1}
+                            max={10}
+                            defaultValue={5}
+                            required
+                        />
+                    </Grid>
+                    <Grid item xs={2}>
+                        <TextField
+                            id="luck"
+                            name="luck"
+                            value={values.luck}
+                            onChange={handleChange}
+                            type="number"
+                            InputProps={{
+                                inputProps: {min: 1, max: 10},
+                                style: {color: 'white'},
+                            }}
+                            variant="outlined"
+                            size="small"
+                        />
+                    </Grid>
+                </Grid>
+            </CardContent>
+            <CardActions>
+                <Button onClick={async () => {
+                    await fetch('http://localhost:5000/', {
+                        method: 'POST',
+                        headers: {
+                            'Content-Type': 'application/json'
+                        },
+                        body: JSON.stringify(values)
+                    })
+                        .then((response) => response.json())
+                        .then((data) => {
+                            console.log(data);
+                            onSubmit(values);
+                        })
+                        .catch((error) => {
+                            console.error('Error when sending a request:', error);
+                        });
+                }}
+                        variant="contained" color="orange">
+                    Generate
+                </Button>
```

Comment:
Need to move fetching data to an external function. Suggest to move URL request to .env file.